openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -1020,7 +1021,9 @@ def post(self, request):
 
         # Render context and response.
         context = self.get_context_data()
-        return self.render_to_response(context)
+        response = self.render_to_response(context)
```

Comment:
1. Especially because we might turn these into exceptions rather than warnings, I wonder if we want a test to ensure this doesn’t regress?
2. Note for future: having this tooling in edx-platform would not allow for plugins with this type of behavior (or extracting this code).

Reply:
I don't understand what you mean by your first point @robrap, is there a more concrete change that you think we should make before merging this?

Reply:
I meant a unit test would ensure we don’t lose this exemption in the future. And, in the future, if we lose the exemption it may start breaking things, because we hope to switch the middleware to raise an exception once we’ve exempted all the false positives.

So, my request is to consider adding a unit test, although that doesn’t need to block this PR.